tests/Support/SupportCollectionTest.php

```diff
@@ -404,6 +404,24 @@ public function testShiftReturnsAndRemovesFirstXItemsInCollection()
         (new Collection(['foo', 'bar', 'baz']))->shift(-2);
     }
 
+    public function testShiftReturnsNullOnEmptyCollection()
+    {
+        $english = new \stdClass();
+        $english->text = 'f';
+        $dutch = new \stdClass();
+        $dutch->text = 'x';
+
+        $translations = collect([$english, $dutch]);
+
+        $englishTranslation = $translations->shift();
+        $dutchTranslation = $translations->shift();
+        $danish = $translations->shift();
+
+        $englishTranslation?->text;
+        $dutchTranslation?->text;
+        $this->assertNull($danish);
```

Comment:
could this be simplified to:?
```suggestion
        $this->assertNull((new Collection())->shift());
```

Reply:
While it could, I feel that the current test is more expressive. As in, it conveys more what it actually tests.
I could reduce the collection to one item, but I feel that's a bit nitpicky